x/stake/client/rest/query.go

```diff
@@ -83,6 +80,62 @@ func bondingStatusHandlerFn(ctx context.CoreContext, storeName string, cdc *wire
 	}
 }
 
+// TODO inherit from Validator
+type StakeValidatorOutput struct {
```

Comment:
Yeah I don't want to maintain two structs now - this should have been fixed to inherit Validator fields before this PR was merged IMO